code/obj/item/fabdisk.dm

```diff
@@ -0,0 +1,29 @@
+obj/item/fabdisk
+	name = "ManuDisk: Empty"
+	desc = "A cartridge for data storage that can be inserted and removed from manufacturers to temporarily add recipes to a manufacturer."
+	icon = 'icons/obj/items/fabdisk.dmi'
+	icon_state = "fabdiskwhi"
+	var/list/datum/manufacture/diskstored = list()
+	var/list/datum/manufacture/disktemp = list()
+	var/fablimit = -1 // Means it can be used unlimited time, its a lazy solution yet an effective one.
```

Comment:
This would make more sense to be on a per-recipe basis I'd think, unless these carts are supposed to be fairly static and mono-recipe

Reply:
Currently these are intended to be static with what recipes they have. I dont intend all recipes to be monorecipe, however there definitely are case usages for when only one recipe would be fitting.

Big issue with per recipe basis is simply put dont really know how to code it, that was what i intended initially. However feel like having to be strategic with how you use your manudisk would be interesting. (Ie Do you want to make another gun or fab more ammo? Becaude doing either would still tick usages down.)

Reply:
Gonna mark as resolved now, if you ultimately want it on a per recipe basis then just realert me.